test/e2e_node/e2e_node_suite_test.go

```diff
@@ -66,6 +66,7 @@ func init() {
 	e2econfig.CopyFlags(e2econfig.Flags, flag.CommandLine)
 	framework.RegisterCommonFlags(flag.CommandLine)
 	framework.RegisterNodeFlags(flag.CommandLine)
+	framework.RegisterClusterFlags(flag.CommandLine)
```

Comment:
I think you misunderstood my previous comment. ClusterFlags are not relevant to the node e2e suite, so they shouldn't be registered here. The flags that are common to both suites (node & cluster) are in `RegisterCommonFlags`. My suggestion was to move the specific flag you need from `RegisterClusterFlags` to `RegisterCommonFlags`

Reply:
Thanks, very clear explanation. Updated